cmd/saga/start/service/service.go

```diff
@@ -68,3 +76,21 @@ func printResults(args saga.CreateLicenseInputArgs, workflowID, runID string) {
 		"temporal.run_id":      runID,
 	}).Info("Successfully completed Workflow")
 }
+
+func injectBaggage(ctx context.Context, workflowID string) (context.Context, error) {
+	// Inject the workflow ID as OTel baggage, so it appears on all spans
+	bgMem, err := baggage.NewMember("foo", "bar")
+	if err != nil {
+		return nil, fmt.Errorf("failed to create otel baggage member: %w", err)
+	}
+	bg, err := baggage.New(bgMem)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create otel baggage: %w", err)
+	}
+
+	// INJECT BAGGAGE!!
+	//ctx = baggage.ContextWithBaggage(ctx, bg)
+	ctx = context.WithValue(ctx, ctxpropagation.PropagateKey, bg)
+
+	return ctx, nil
+}
```

Comment:
Here we are in the Temporal workflow starter. We create some OTel baggage and inject it in context using our special `PropagateKey`.